lib/util/errors.js

```diff
@@ -33,8 +33,15 @@ const
  * @param  {string} subdomain - Subdomain (eg: 'elasticsearch')
  * @param  {string} error - Error name: (eg: 'index_not_found')
  * @param  {...any} placeholders - Placeholders value to inject in error message
+ * @param  {object} options - Last param can be additional options { message }
  */
 function get (domain, subdomain, error, ...placeholders) {
+  let options = {};
+  // @todo test this
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS1135)